File: executable/julia/Electric-heatmap.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = sqrt(r*c/size^2)

data = [
  [
    "z" => z, 
    "scl" => "Electric", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "Electric"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "Electric-heatmap", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/r/Electric-heatmap.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- sqrt(r*c/size^2)
    }
}

data <- list(
  list(
    z = z, 
    scl = "Electric", 
    type = "heatmap"
  )
)
layout <- list(title = "Electric")

response <- p$plotly(data, kwargs=list(layout=layout, filename="Electric-heatmap", fileopt="overwrite"))
url <- response$url
filename <- response$filename